README.md

```diff
@@ -102,503 +102,77 @@ identifiable. If you are prototyping, you can use `java.util.UUID` to generate a
+    // Subscribe to incoming messages
+    val messageSubscription = room.messages.subscribe { event ->
+        println("Message received: ${event.message.text}")
+    }
 
-```kotlin
-room.reactions.send(
-    type ="like",
-    metadata = JsonObject().apply { addProperty("effect", "fireworks") },
-    headers = mapOf("streamId" to "basketball-stream"),
-)
-```
+    // Attach the room - meaning we'll start receiving events from the server
+    room.attach()
 
-### Subscribing to room reactions
+    // Send a message
+    room.messages.send(text = "Hello, World! This is my first message with Ably Chat!")
 
-Subscribe to receive room-level reactions:
+    // Wait for 5 seconds to make sure the message has time to be received.
+    delay(5000)
 
-```kotlin
-val subscription = room.reactions.subscribe { reaction: ReactionEvent ->
-    println("received a ${reaction.type} with metadata ${reaction.metadata}")
+    // Now release the room and close the connection
+    chatClient.rooms.release("readme-getting-started")
+    realtimeClient.close()
 }
 ```
 
-### Unsubscribing from room reactions
+All being well, you should see lines in logcat resembling the following:
 
-To unsubscribe the listener, you can call the `unsubscribe` method on the subscription object returned by the `subscribe` call:
-
-```kotlin
-val subscription = room.reactions.subscribe { reaction: ReactionEvent ->
-    println("received a ${reaction.type} with metadata ${reaction.metadata}")
-}
-
-// Time passes...
-subscription.unsubscribe()
 ```
-
-## In-depth
-
-### Channels Behind Chat Features
-
-It might be useful to know that each feature is backed by an underlying Pub/Sub channel. You can use this information to enable
-interoperability with other platforms by subscribing to the channels directly using
-the [Ably Pub/Sub SDKs](https://ably.com/docs/products/channels) for those platforms.
-
-The channel for each feature can be obtained via the `channel` property
-on that feature.
-
-```kotlin
-val messagesChannel = room.messages.channel
+Connection status changed: Connected
+Room status changed: Attaching
+Room status changed: Attached
+Message received: Hello, World! This is my first message with Ably Chat!
+Room status changed: Releasing
+Room status changed: Released
 ```
 
-**Warning**: You should not attempt to change the state of a channel directly. Doing so may cause unintended side-effects in the Chat SDK.
-
-### Channels Used
-
-For a given chat room, the channels used for features are as follows:
-
-| Feature   | Channel                              |
-|-----------|--------------------------------------|
-| Messages  | `<roomId>::$chat::$chatMessages`     |
-| Presence  | `<roomId>::$chat::$chatMessages`     |
-| Occupancy | `<roomId>::$chat::$chatMessages`     |
-| Reactions | `<roomId>::$chat::$reactions`        |
-| Typing    | `<roomId>::$chat::$typingIndicators` |
-
----
+Congratulations! You have sent your first message using the Ably Chat SDK!
 
```

Comment:
Can we add a section that points to official documentation `https://ably.com/docs/chat`

```suggestion
The current documentation provides basic instructions for setup and sending a message. For more detailed documentation, visit https://ably.com/docs/chat.
```

Reply:
We do have link to official doc at the start of the README, but it's hard to notice and it will be great if we have explicit note for the same saying official doc exists on a separate link.

Reply:
[c7e858a](https://github.com/ably/ably-chat-kotlin/pull/104/commits/c7e858a740dcf35e1874fcccdddec70f61883355)